### ISSUE TITLE
Installation fails on Mac / Python 3.10

### ISSUE THREAD

**Issue by haesleinhuepf** [xcube-dev/xcube#1027]
**Describe the bug**

Hi, I'm trying to install xcube in a conda environment on MacOS Sonoma 14.5 with Python 3.10 running.

**To Reproduce**

I run this command (from [here](https://xcube.readthedocs.io/en/latest/installation.html)):
```
mamba install --channel conda-forge xcube
```

That's the error message:
```
Looking for: ['xcube']

pkgs/main/osx-64         [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) No change
pkgs/r/osx-64            [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change
conda-forge/noarch       [====================] (00m:15s) Done
conda-forge/osx-64       [====================] (00m:24s) Done
Encountered problems while solving.
Problem: nothing provides _python_rc needed by python-3.12.0rc3-rc3_h30d4d87_1_cpython
```

**Expected behavior**

The installation works.

**Screenshots**

**Additional context**

Let me know if you need any further info!

**Comment by forman** [comment]
Hi @haesleinhuepf, this is strange as the build server succeeds on MacOS. Could you please try again?

**Comment by haesleinhuepf** [comment]
Hi @forman ,

thanks for the feedback I just tried with the command above and got the same error. It was a bit confusing because the error messag mentiones python 3.12. 

Hence I tried again using this command in the base environment and it worked:
```
mamba create --name test python=3.10 xcube
```

I'm closing this issue. I presume this was an error related to my environment.

Thanks again!

Best,
Robert
